src/systems/ec/implementations/react/components/file/src/FileDownload.jsx

```diff
@@ -62,42 +108,99 @@ export const FileDownload = ({
           />
 
           <ul className="ecl-file__translation-list">
-            {translation.items.map(item => (
-              <li
-                className={classnames(
-                  item.className,
-                  'ecl-file__translation-item'
-                )}
-                key={item.title}
-              >
-                <div className="ecl-file__translation-info">
-                  <div
-                    className="ecl-file__translation-title"
-                    {...(item.lang && { lang: item.lang })}
+            {variant === 'thumbnail' ? (
+              <Fragment>
+                {translation.items.map(item => (
+                  <li
+                    className={classnames(
+                      item.className,
+                      'ecl-file__translation-item'
+                    )}
+                    key={item.title}
+                  >
+                    <div className="ecl-file__translation-detail">
+                      <div
+                        className="ecl-file__translation-title"
+                        {...(item.lang && { lang: item.lang })}
+                      >
+                        {item.title}
+                      </div>
+                      <div
+                        className="ecl-file__translation-description"
+                        {...(item.lang && { lang: item.lang })}
+                      >
+                        {item.description}
+                      </div>
+                    </div>
+                    <div className="ecl-file__translation-info">
+                      <div className="ecl-file__translation-language">
+                        {item.langFull}
+                      </div>
+                      <div className="ecl-file__translation-meta">
+                        {item.meta}
+                      </div>
+                    </div>
+                    <Link
+                      {...item.download}
+                      icon={{
+                        shape: 'ui--download',
+                        size: 'fluid',
+                      }}
+                      href={item.download.href || download.href}
+                      label={item.download.label || download.label}
+                      variant="standalone"
+                      className={classnames(
+                        item.download.className,
+                        'ecl-file__translation-download'
+                      )}
+                      download
+                      {...(item.lang && { hrefLang: item.lang })}
+                    />
+                  </li>
```

Comment:
Bottom border of last item of the translation list is not 100% width:

<img width="815" alt="Screenshot 2020-07-08 at 09 17 57" src="https://user-images.githubusercontent.com/1289480/86889705-770f3b80-c0fc-11ea-9b4a-7114bd4691a2.png">